perf/alloc_per_constructor.jl

```diff
@@ -3,6 +3,18 @@ import Pkg
 Pkg.develop(path = ".")
 import Profile
 
+using TOML
+planet_parse = TOML.parsefile(joinpath("test", "planet_parameters.toml"))
+param_dict = Dict{Symbol, Any}()
+for (key, val) in planet_parse
+    # In the future - we will use the full names,
+    # param_dict[Symbol(key)] = val["value"]
+
+    # for now we use the aliases
+    param_dict[Symbol(val["alias"])] = val["value"]
+end
+full_parameter_set = (; param_dict...)
```

Comment:
Maybe this should just go in `common.jl`?

Reply:
Yeah i see now that this is required in the benchmark tests etc too